op-node/rollup/derive/span_batch_txs.go

```diff
@@ -91,9 +91,9 @@ func (btx *spanBatchTxs) decodeContractCreationBits(r *bytes.Reader) error {
 	return nil
 }
 
-func (btx *spanBatchTxs) contractCreationCount() uint64 {
+func (btx *spanBatchTxs) contractCreationCount() (uint64, error) {
 	if btx.contractCreationBits == nil {
-		panic("contract creation bits not set")
+		return 0, errors.New("dev error: contract creation bits not set")
```

Comment:
https://github.com/ethereum-optimism/optimism/blob/c829e0cb702ee8b76ca6d28029dadf5d2c2f2068/op-node/rollup/derive/channel.go#L138

I tried to follow this error message because if that error is raised, it is clearly an dev error. The method `contractCreationCount` is expected to be called after `decodeContractCreationBits` method is called.